### ISSUE TITLE
pip install version 0.0.9: cannot find CHANGELOG.rst

### ISSUE THREAD

**Issue by Enaero** [allelos/vectors#1]
I am having trouble pip installing vectors 0.0.9:

```
C:\Users\billy>pip install vectors
Collecting vectors
  Using cached vectors-0.0.9.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\billy\AppData\Local\Temp\pip-build-2zfgdkmc\vectors\setup.py", line 14, in <module>
        long_description=read('CHANGELOG.rst'),
      File "C:\Users\billy\AppData\Local\Temp\pip-build-2zfgdkmc\vectors\setup.py", line 5, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
    FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\billy\\AppData\\Local\\Temp\\pip-build-2zfgdkmc\\vectors\\CHANGELOG.rst'
```

It works with 0.0.8:

```
C:\Users\billy>pip install vectors==0.0.8
Collecting vectors==0.0.8
  Downloading vectors-0.0.8.tar.gz
Installing collected packages: vectors
  Running setup.py install for vectors ... done
Successfully installed vectors-0.0.8
```

**Comment by SashaZd** [comment]
I can report I have the same error with the 0.0.9

**Comment by allelos** [comment]
Fixed it with version 1.0.0. Should work fine.
